fbw-common/src/systems/datalink/router/src/webinterfaces/HoppieConnector.ts

```diff
@@ -388,6 +388,8 @@ export class HoppieConnector {
     }
 
     public static pollInterval(): number {
-        return 5000;
+        // To comply with Hoppie rate limits, we choose a random number between 45 and 75, as recommend by Hoppie. Ref to: https://www.hoppie.nl/acars/system/tech.html
+        const pollTimeSeconds = Math.floor(Math.random() * (75 - 45 + 1)) + 45;
+        return pollTimeSeconds * 1000;
```

Comment:
```suggestion
        return Math.random() * 30000 + 45000;
```

Reply:
Agreed, good idea.

Reply:
Oh yeah for sure can one-liner it, didn't think of approaching it like that.

Though I have to ask, what's the reason for increasing the poll rate? I am following the recommendation of the network itself for the 75-45 rate.

Reply:
That code would give you a poll interval between 0 + 45_000 ms and 30_000 + 45_000 = 75_000 ms, so between 45 and 75 seconds.